mathesar_ui/src/stores/table-data/columns.ts

```diff
@@ -215,4 +216,8 @@ export class ColumnsDataStore implements Writable<ColumnsData> {
     this.promise = null;
     this.listeners.clear();
   }
+
+  deleteColumn(columnId:number): Promise<void> {
+    return deleteAPI(`${this.url}${columnId}/`);
```

Comment:
I added a return statement to follow the same pattern in the add column function. But not sure why it's necessary. They would still work without it.

Reply:
Thanks for the explanation @seancolsen! That makes sense.